meetings/meetings/settings/local-dist.py

```diff
@@ -5,6 +5,10 @@
 DEBUG = True
 CLIENT_ID = ''
 CLIENT_SECRET = ''
+MAILGUN_API_KEY = ''
+ANYMAIL = {
+    "MAILGUN_API_KEY": MAILGUN_API_KEY,
+}
```

Comment:
Put these in defaults.py and comment that they need to be completed in local.py.